src/Component.php

```diff
@@ -204,8 +206,10 @@ private function handleFor( DOMNode $node, array $data ) {
 		if ( $node->hasAttribute( 'v-for' ) ) {
 			list( $itemName, $listName ) = explode( ' in ', $node->getAttribute( 'v-for' ) );
 			$node->removeAttribute( 'v-for' );
+			$node->removeAttribute( ':key' );
```

Comment:
This looks unrelated… I could live with including it here but it should at least have a test ^^

Reply:
Fair. I've added a test.